src/devices/sprut.ts

```diff
@@ -0,0 +1,117 @@
+import * as fz from "../converters/fromZigbee";
```

Comment:
Also add this file to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/index.ts